aicsimageio/tests/expectations.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Was this meant to be committed? I don't see a reference to this file anywhere...

Reply:
whoops... nope sorry. something I was playing with.  perhaps for a later PR :)

Reply:
I think I know what you were trying to do and ya..... we definitely need a central file where all these expectations are kept in cases like this were multiple readers or tests are doing largely the same checks. :joy:

Reply:
Later PR though.